src/engine/guidance/post_processing.cpp

```diff
@@ -608,7 +608,7 @@ std::vector<RouteStep> buildIntersections(std::vector<RouteStep> steps)
     {
         auto &step = steps[step_index];
         const auto instruction = step.maneuver.instruction;
-        if (instruction.type == TurnType::Suppressed)
+        if (instruction.type == TurnType::Suppressed || instruction.type == TurnType::UseLane)
```

Comment:
There is a slight difference in how we treat these instructions in multiple processing steps. I can try, but I am unsure how it would interfere with other items (e.g. lane anticipation and or collapsing). Will report back after trying.